tasks/modules.py

```diff
@@ -54,5 +55,9 @@ def dependency_path(self, agent_version):
 
 
 DEFAULT_MODULES = {
-    ".": GoModule(".", targets=["./pkg", "./cmd"]),
+    ".": GoModule(".", targets=["./pkg", "./cmd"], dependencies=["pkg/util/log", "pkg/util/winutil"]),
+    "pkg/util/log": GoModule("pkg/util/log"),
+    "pkg/util/winutil": GoModule(
+        "pkg/util/winutil", condition=lambda: sys.platform == 'win32', dependencies=["pkg/util/log"]
```

Comment:
Shouldn't the `+build` exclude the need for any explicit dependency exclusion/inclusion?

Reply:
The `dependencies` field is used by #7527 to update the `require` directives on the different `go.mod` files.

For example, when we release `7.28` and the release manager runs the `inv release.finish` in the main `go.mod` file this line
```
 	github.com/DataDog/datadog-agent/pkg/util/log v0.0.0
```

will be replaced by
```
 	github.com/DataDog/datadog-agent/pkg/util/log v0.28.0
```

This is briefly mentioned here https://github.com/DataDog/mars-jenkins-scripts/blob/master/release_documentation/final_artifacts.md#updating-the-releasejson-and-module-dependencies-release-coordinator